### PR TITLE
Add x-kc-source header to sdk calls to be able to see the usage of th…

### PR DESCRIPTION
**Pull request by JiriLojda**
### Motivation

Track sdk calls from this integration to be able to see the real usage and prioritize based on that.

### Checklist

- [x] Code follows coding conventions held in this repo
- [x] Docs/README have been updated (if applicable)
- [x] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,22 +1,11 @@
...
     "@kontent-ai/delivery-sdk": "^12.1.0",
     "@kontent-ai/webhook-helper": "^2.0.1",
     "@netlify/functions": "^1.2.0"
   },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/kontent-ai/netlify-webhook-processor.git"
-  },
-  "author": "strizr",
-  "license": "MIT",
```
Comment (Simply007) [inline package.json:15]: put the License back

File: package.json [hunk 2]
```diff
@@ -1,22 +1,11 @@
...
-  "bugs": {
-    "url": "https://github.com/kontent-ai/netlify-webhook-processor/issues"
-  },
-  "homepage": "https://github.com/kontent-ai/netlify-webhook-processor#readme",
```
Comment (Simply007) [inline package.json:19]: put homepage+bugy back

File: package.json [hunk 3]
```diff
@@ -1,22 +1,11 @@
...
-  "name": "netlify-webhook-processor",
+  "name": "@kontent-ai/integration-example-netlify",
   "version": "1.0.0",
-  "description": "",
   "dependencies": {
     "@kontent-ai/delivery-sdk": "^12.1.0",
     "@kontent-ai/webhook-helper": "^2.0.1",
     "@netlify/functions": "^1.2.0"
   },
-  "repository": {
```
Comment (Simply007) [inline package.json:10]: put repository back

File: functions/kontent_ai_webhook_generic.ts [hunk 4]
```diff
@@ -43,7 +44,10 @@ export const handler: Handler = async (event) => {
   }
 
   // download the affected items
-  const deliveryClient = new DeliveryClient({ projectId });
+  const deliveryClient = new DeliveryClient({
+    projectId,
+    globalHeaders: () => [{ header: 'X-KC-SOURCE', value: `${packageJson.name};${packageJson.version}` }]
```
Comment (Simply007) [inline functions/kontent_ai_webhook_generic.ts:49]: I might add the parameter with underscore to see that original headers are present in `_headers`

**Review by Simply007 — CHANGES_REQUESTED** [review]

File: functions/kontent_ai_webhook_generic.ts (hunk 4 above)
Comment (JiriLojda) [reply functions/kontent_ai_webhook_generic.ts:49]: same reason as here: https://github.com/kontent-ai/integration-example-recombee/pull/6#discussion_r1073425158

**Review by Simply007 — APPROVED** [review]

**Pull request merged**